website/src/components/pageStateSelectors/VariantSelector.tsx

```diff
@@ -6,9 +6,11 @@ import { GsMutationFilter } from '../genspectrum/GsMutationFilter.tsx';
 export function VariantSelector({
     onVariantFilterChange,
     variantFilterConfig,
+    unaligned,
```

Comment:
I don't like the name of this prop - I would rather call it something like "hide mutations filter" or similar. Or the version that doesn't require double negations (with a `true` default value):

```suggestion
    showMutationFilter = true,
```